### PR TITLE
Multi-website Support: Fix getApiKey value, webhooks.

### PR DESCRIPTION
**Pull request by YevhenShBolt**

### CODE REVIEW

File: Model/Api/ShippingMethods.php [hunk 1]
```diff
@@ -313,7 +313,8 @@ public function getShippingMethods($cart, $shipping_address)
         try {
 //            $this->logHelper->addInfoLog($this->request->getContent());
 
-            $this->preprocessHook();
+            // TODO: Temporally comment out.
+            //$this->preprocessHook();
```
Comment (danac-gs) [inline Model/Api/ShippingMethods.php]: you probably need to remove this now

Comment (danac-gs) [inline Model/Api/ShippingMethods.php]: Delete these lines?

**Comment by matt-thomason** [comment]
@YevhenShBolt Have you done the testing on single-store setup as @danac-gs asked?

File: Helper/Order.php [hunk 2]
```diff
@@ -1443,4 +1457,60 @@ protected function getVoidMessage(OrderPaymentInterface $payment)
...
+        return ($order && $order->getStoreId()) ? $order->getStoreId() : null;
+    }
+
+    /**
+     * @param string $incrementId
+     * @param bool   $forceLoad
+     *
+     * @return OrderInterface
+     */
+    public function getOrderByIncrementId($incrementId, $forceLoad = false)
```
[the remark this answers is not in the document]
Comment (YevhenShBolt) [reply Helper/Order.php]: Added description.

File: Model/Api/ShippingMethods.php (hunk 1 above)
Comment (YevhenShBolt) [reply Model/Api/ShippingMethods.php]: removed

**Comment by YevhenShBolt** [comment]
> @YevhenShBolt Have you done the testing on single-store setup as @danac-gs asked?
@matt-thomason yes, i checked. https://app.asana.com/0/0/1124794684851746/f

**Comment by matt-thomason** [comment]
@YevhenShBolt Is this still [WIP]?

**Pull request merged**